lblod/ember-rdfa-editor-lblod-plugins
# GN-4449: table of contents internationalization

### Overview
This PR introduces internationalization of the table of contents nodes based on the current document language.
It adds the following:
- Internationalization of the dynamic version of the table of contents based on the document language and using ember-intl.
- Internationalization of the static version (in the `serialize` method) of the table of contents based on the document language.
   * Note: the `emberApplication` prosemirror plugin is required for the internationalization to work (the `serialize` method needs to be able to access the ember-intl service), otherwise the static version will default to dutch.
- Additionally, this PR ensures that the table-of-contents is always correctly exported in the `serialize` method based on the current prosemirror state. The `entries` attribute of the table-of-contents node is no longer required.

##### connected issues and PRs:
Main internationalization ticket: [GN-4448](https://binnenland.atlassian.net/browse/GN-4448?atlOrigin=eyJpIjoiYzAwYzJiNzM5MDkyNDU3NmE1NzIxZjMyZWIxNjlkZWEiLCJwIjoiaiJ9)
Table of contents internationalization ticket: [GN-4449](https://binnenland.atlassian.net/browse/GN-4449?atlOrigin=eyJpIjoiMzkyNjZhZmUyNzdlNDRkYWJhY2VjODQxZGJkNzFkNWMiLCJwIjoiaiJ9)

### How to test/reproduce
- Insert the table of contents of a regulatory statement.
- Ensure the table of contents is translated in dutch, the exported version will also be displayed in dutch. Note that the table of contents node is now completely serialized (and no longer only it's title).
- Change the document language in the console (e.g. `window.__PC.documentLanguage = 'en-US'`).
- Notice the language of the dynamic table of contents change.
- Check if the exported version of the table of contents is now correctly translated.



### Checks PR readiness
